### PR TITLE
Add nixos-21.11 to the build matrix

### LINKED ISSUE #5: Provide a 21.11 release?
It would be nice to provide a 21.11 release for netboot.xyz to use.

### PR DESCRIPTION
**Pull request by nshalman**
Fixes #5

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ jobs:
         tag:
           - nixos-20.09
```
Comment (Mic92) [inline .github/workflows/build.yml:12]:
```suggestion
```

**Comment by nshalman** [comment]
Will removing 20.09 delete the last existing release? Will that break netboot.xyz?

**Comment by Mic92** [comment]
It's old and I don't think there will be any (security) updates.

**Comment by Mic92** [comment]
> Will removing 20.09 delete the last existing release? Will that break netboot.xyz?

No. There will be no more updates though.

**Pull request merged**